### DISCUSSION TITLE
删除团队后，团队里项目所属的domain还在怎么办？

### DISCUSSION THREAD

**Discussion by suyinLoong** [vercel/community#141] [Help] [answered]
最近删除了一个团队，团队里有一个项目，项目里所属的domain并没有特意删除，结果现在我的新项目想要使用那个已被删除的团队里所属项目的domain时，发现那个团队虽然被删除了，但是其下的项目中的domain并没有被移除，这就导致了那个domain是占用状态而无法被重新使用，该怎么办？
![error](https://user-images.githubusercontent.com/49554285/141037620-d502ef50-ecec-474e-8ebc-82e838366857.png)

**Comment by suyinLoong** [comment, marked as answer]
已解决！

**Reply by Pathsis** [reply]
请问您如何解决的。我没有先移除域名而直接删除Vercel的项目，现在访问我的域名，总是**间歇性地**收到来自Vercel的404错误页面。我确定我的域名已经跟Vercel毫无关系了。

**Reply by suyinLoong** [reply]
> 请问您如何解决的。我没有先移除域名而直接删除Vercel的项目，现在访问我的域名，总是**间歇性地**收到来自Vercel的404错误页面。我确定我的域名已经跟Vercel毫无关系了。

据官网文档说，删除账号一段时间再注册就可以了，具体是多久我忘记了，你可以查看一下官方文档，目前我试过这个方法，可能是删除的时间不够，没有生效，重新注册后以前的域名依旧存在。

**Reply by Pathsis** [reply]
谢谢。一个教训是，不要在移除域名之前删除项目或团队。
